modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/test/js/components/export-form/ExportFormModal.js

```diff
@@ -0,0 +1,159 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {act, fireEvent, render, screen} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import {ExportFormModal} from 'dynamic-data-mapping-form-web/admin/js/components/export-form/openExportFormModal.es';
```

Comment:
Please, use relative path:
```suggestion
import {ExportFormModal} from ''../../../../src/main/resources/META-INF/resources/admin/js/components/export-form/openExportFormModal.es';
```
It helps on code development, enabling IntelliSense, and avoid the need to a dependency bundled to perform the tests